### ISSUE TITLE
Default limits result in exception

### ISSUE THREAD

**Issue by jwmarshall** [confluentinc/cp-docker-images#142]
```
1===> Launching zookeeper ...
[2016-09-29 01:58:16,431] INFO Reading configuration from: /etc/kafka/zookeeper.properties (org.apache.zookeeper.server.quorum.QuorumPeerConfig)
[2016-09-29 01:58:16,602] ERROR Invalid config, exiting abnormally (org.apache.zookeeper.server.quorum.QuorumPeerMain)
org.apache.zookeeper.server.quorum.QuorumPeerConfig$ConfigException: Error processing /etc/kafka/zookeeper.properties
	at org.apache.zookeeper.server.quorum.QuorumPeerConfig.parse(QuorumPeerConfig.java:123)
	at org.apache.zookeeper.server.quorum.QuorumPeerMain.initializeAndRun(QuorumPeerMain.java:101)
	at org.apache.zookeeper.server.quorum.QuorumPeerMain.main(QuorumPeerMain.java:78)
Caused by: java.lang.NumberFormatException: For input string: "5)"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:580)
	at java.lang.Integer.parseInt(Integer.java:615)
	at org.apache.zookeeper.server.quorum.QuorumPeerConfig.parseProperties(QuorumPeerConfig.java:159)
	at org.apache.zookeeper.server.quorum.QuorumPeerConfig.parse(QuorumPeerConfig.java:119)
	... 2 more
Invalid config, exiting abnormally
```

https://github.com/confluentinc/cp-docker-images/blob/master/debian/zookeeper/include/etc/confluent/docker/configure#L31-L32

The `)` character needs to be removed from the above lines.

**Comment by samjhecht** [comment]
Thanks for raising this @jwmarshall .  I just pushed a fix and @arrawatia will provide an update here once we've uploaded corrected versions of the image.

**Comment by jwmarshall** [comment]
👍

**Comment by samjhecht** [comment]
Closing this issue.  fixed in 3.0.1 2 docker images
